src/accelerate/utils/deepspeed.py

```diff
@@ -234,7 +234,7 @@ class DummyOptim:
             parameter groups
         weight_decay (float):
             Weight decay.
-        **kwargs:
+        **kwargs (additional keyword arguments, *optional*):
```

Comment:
Does the `**` make a difference?

Reply:
Nope, not in terms of formatting! `**kwargs` still shows up as expected. We can remove the `**` if you want :)

<img width="830" alt="Screenshot 2024-03-26 at 12 39 09 PM" src="https://github.com/huggingface/accelerate/assets/59462357/9231a285-144c-4b4e-9bab-dc6e3807142b">

Reply:
I got you 👍